releases/release-1.31/release-notes/maps/pr-124131-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 124131
+releasenote:
+  text: Endpointslices mirrored from Endpoints by the EndpointSliceMirroring controller
```

```suggestion
  text: Fixed the bug where if Endpointslices mirrored from Endpoints by the EndpointSliceMirroring controller they would not reconcile if modified.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653201734